mailpoet/lib/EmailEditor/Engine/Renderer/template.html

```diff
@@ -62,6 +62,7 @@
                   border="0"
                   cellpadding="0"
                   cellspacing="0"
+                  style="width: 100%"
```

Comment:
👍 Good catch!
During the review and testing, I noticed that this table does not contain table tags the row and the column.
Will we create a ticket, or will you take a look?